src/main/java/com/ipcounter/classes/FileProcessor.java

```diff
@@ -0,0 +1,88 @@
+package com.ipcounter.classes;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+public class FileProcessor extends Thread {
+
+  private long currentLine = 0;
+  private final Path path;
+  private IOException exception = null;
+  CounterStorage storage;
+
+  /**
+   * Creates a thread that is ready to run on a given file
+   * 
+   * @param filePath relative or absolute path to the file
+   * @throws IOException if file does not exist
+   */
+  public FileProcessor(final String filePath) throws IOException {
+    this.path = Paths.get(filePath);
+    if (!Files.exists(path)) {
+      throw new IOException(
+          String.format(
```

Comment:
can be in one line.

Reply:
It can be saved in one variable and value of variable can be put into method. It is not JavaScript or Python for this strange formatting.